concepts/team-changenotification-channels.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Get change notifications for change in creation, updation or deletion of channels using Microsoft Graph"
+description: "Get change notifications for change in creation, updation or deletion of channels using Microsoft Graph"
+author: "AbhishekAnand"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for channel using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to a [channel](/graph/api/resources/channel?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
+
+>**Note:** The maximum time a subscription can last is 60 minutes; however, subscriptions can be renewed until the caller has permissions to access to resource.
+
+
+
+### Subscribe to changes in any channel at tenant level
+
+To get change notifications for any change in channel's property across any channel in tenant, subscribe to `teams/getAllChannels`. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification.
```

Comment:
this statement should specify not just change in channel's property, but creation/deletion too